controller.js

```diff
@@ -59,7 +59,7 @@ async function getCommentsByArticleId (req, res, next) {
     ]
     Promise.all(promises).then((resolutions) => {
         if (resolutions[0].length === 0) {
-            return Promise.reject({status: 404, msg: "No comments found"})
+            return Promise.reject({status: 200, msg: "No comments found"})
```

Comment:
so we won't reject with a 200, as this is not an error - we want to send an empty array here and the body, to show there are no comments for this article yet